### PR TITLE
PLAT-324: remove old insolar.Reference/ID

### PR DESCRIPTION
**Pull request by sexyhamster**
remove old insolar.Reference/ID in favour of reference.Global/Local

### CODE REVIEW

File: ledger-core/v2/application/genesis.go [hunk 1]
```diff
@@ -7,21 +7,23 @@ package application
 
 import (
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/longbits"
 )
 
-type genesisBinary []byte
+type genesisBinary longbits.Bits224
```
Comment (cyraxred) [inline ledger-core/v2/application/genesis.go:14]: please use reference.LocalHash instead an explicit type

File: ledger-core/v2/insolar/jet/tree_test.go [hunk 2]
```diff
@@ -23,7 +25,7 @@ func TestTree_Update(t *testing.T) {
 		prefix []byte
 	)
 
-	lookup := insolar.NewID(gen.PulseNumber(), []byte{0xD5}) // 11010101
+	lookup := reference.NewRecordID(gen.PulseNumber(), longbits.Bits224{0xD5}) // 11010101
```
Comment (cyraxred) [inline ledger-core/v2/insolar/jet/tree_test.go:28]: please use reference.LocalHash

File: ledger-core/v2/insolar/jet/tree_test.go [hunk 3]
```diff
@@ -70,7 +72,7 @@ func TestTree_Find(t *testing.T) {
 			},
 		},
 	}
-	lookup := insolar.NewID(gen.PulseNumber(), []byte{0xD5}) // 11010101
+	lookup := reference.NewRecordID(gen.PulseNumber(), longbits.Bits224{0xD5})
```
Comment (cyraxred) [inline ledger-core/v2/insolar/jet/tree_test.go:75]: please use reference.LocalHash

File: ledger-core/v2/insolar/store/jetindex_test.go [hunk 4]
```diff
@@ -41,9 +43,9 @@ func TestJetIndex_For(t *testing.T) {
 	t.Parallel()
 
 	idx := NewJetIndex()
-	id := insolar.NewID(insolar.PulseNumber(4), []byte{1})
-	sID := insolar.NewID(insolar.PulseNumber(4), []byte{2})
-	tID := insolar.NewID(insolar.PulseNumber(4), []byte{3})
+	id := reference.NewRecordID(insolar.PulseNumber(4), longbits.Bits224{1})
```
Comment (cyraxred) [inline ledger-core/v2/insolar/store/jetindex_test.go:46]: please use reference.LocalHash

File: ledger-core/v2/application/genesis.go [hunk 5]
```diff
@@ -7,21 +7,23 @@ package application
...
 
 // GenesisRecord is initial chain record.
-var GenesisRecord genesisBinary = []byte{0xAC}
+var GenesisRecord = genesisBinary(longbits.Bits224{0xAC})
 
 // ID returns genesis record id.
-func (r genesisBinary) ID() insolar.ID {
-	return insolar.NewID(insolar.GenesisPulse.PulseNumber, r)
+func (r genesisBinary) ID() reference.Local {
+	return reference.NewRecordID(insolar.GenesisPulse.PulseNumber, longbits.Bits224(r))
```
Comment (cyraxred) [inline ledger-core/v2/application/genesis.go:21]: please use reference.LocalHash

File: ledger-core/v2/application/genesis.go [hunk 6]
```diff
@@ -7,21 +7,23 @@ package application
...
+	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/longbits"
 )
 
-type genesisBinary []byte
+type genesisBinary longbits.Bits224
 
 // GenesisRecord is initial chain record.
-var GenesisRecord genesisBinary = []byte{0xAC}
+var GenesisRecord = genesisBinary(longbits.Bits224{0xAC})
```
Comment (cyraxred) [inline ledger-core/v2/application/genesis.go:17]: please use reference.LocalHash

**Comment by sexyhamster** [comment]
fixed longbits.Bits224 usage

**Pull request merged**